packages/storybook/src/stories/grid-rendering.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { ElementSize, XGrid } from '@material-ui/x-grid';
+import '../style/grid-stories.css';
+
+export default {
+  title: 'X-Grid Tests/Rendering',
+  component: XGrid,
+  parameters: {
+    options: { selectedPanel: 'storybook/storysource/panel' },
+    docs: {
+      page: null,
+    },
+  },
+};
+export const RenderInputInCell = () => {
+  const size: ElementSize = { width: 800, height: 600 };
+
+  const handleInputKeyDown = () => {};
```

Comment:
How about we log the input value in action?

Reply:
done